keras/utils/training_utils.py

```diff
@@ -4,9 +4,8 @@
 from __future__ import division
 from __future__ import print_function
 
```

Comment:
why delete the imports

Reply:
Which ones? The lines you point to are not deletions.

The only deletion near there is to move importing layers.merge.concatenate down to avoid a circular import issue.